Sources/Controllers/2.0/RoutePlanning/OABaseTableViewController.h

```diff
@@ -13,7 +13,10 @@
 @property (strong, nonatomic) IBOutlet UIView *navbarView;
 @property (strong, nonatomic) IBOutlet UIButton *backButton;
 @property (strong, nonatomic) IBOutlet UIButton *cancelButton;
+@property (strong, nonatomic) IBOutlet UIButton *doneButton;
 @property (strong, nonatomic) IBOutlet UITableView *tableView;
 @property (strong, nonatomic) IBOutlet UILabel *titleLabel;
 
+- (IBAction)doneButtonPressed:(id)sender;
```

Comment:
It is not the best practice to expose the outlets. You should make the [self dismissViewController]; public and call it instead